gdk-pixbuf/sys/src/lib.rs

```diff
@@ -93,6 +184,98 @@ impl ::std::fmt::Debug for GdkPixbufLoaderClass {
     }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct GdkPixbufModule {
+    pub module_name: *mut c_char,
+    pub module_path: *mut c_char,
+    pub module: gpointer,
```

Comment:
OOC, can you make use of all this without having `gmodule-sys` bindings?

Reply:
Yes, pixbuf loaders do not need to poke at that field, they just need to fill the vtable

Reply:
Ok